DESCRIPTION

```diff
@@ -26,7 +26,7 @@ URL: http://restrserve.org, https://github.com/rexyai/RestRserve
 BugReports: https://github.com/rexyai/RestRserve/issues
 License: GPL (>= 2)
 Depends:
-  R (>= 3.6.0)
+  R (>= 3.0.0)
```

Comment:
Let's move this change to a separate PR in order to not mix things. If there is a demand to downgrade dependency, let's try to:

1. depending on the R version either use errorCondition from base or use version from backports
1. add r 3.0.* environment to the travis and appveyor CI in order to make sure things work with this version